vcd/resource_vcd_vapp_network_test.go

```diff
@@ -139,7 +139,7 @@ func isVappNetworkFound(conn *VCDClient, rs *terraform.ResourceState) (bool, err
 
 	var found bool
 	for _, vappNetworkConfig := range networkConfig.NetworkConfig {
-		if vappNetworkConfig.NetworkName == newVappNetworkName && vappNetworkConfig.Configuration.IPScopes.IPScope.DNSSuffix == dnsSuffix {
+		if vappNetworkConfig.NetworkName == newVappNetworkName && vappNetworkConfig.Configuration.IPScopes.IPScope[0].DNSSuffix == dnsSuffix {
```

Comment:
Why have we made this an array, but using only the first occurrence? Why not just use the single instance of IPScope if we don't need the array?

Reply:
Please check comment above.

Reply:
Thanks for filing #235 about this.